cmd/containerd-stress/main.go

```diff
@@ -149,6 +149,11 @@ func main() {
 			Usage: "set the runtime to stress test",
 			Value: plugin.RuntimeRuncV2,
 		},
+		cli.StringFlag{
+			Name:  "snapshotter",
+			Usage: "set the snapshotter to use",
+			Value: "overlayfs",
+		},
```

Comment:
Why no effect on `density` subcommand?

Reply:
Added it now. Thanks for pointing out.

Reply:
Not sure why this test is failing "CI / Project Checks". Other than that do you have any comment?  @ktock

Reply:
CI looks green now.